contracts/cw1-subkeys/src/contract.rs

```diff
@@ -1253,7 +1243,7 @@ mod tests {
         let spender2 = HumanAddr::from("spender0002");
         let denom = "token1";
         let amount = 10000;
-        let allow = coin(amount, denom);
+        let coin = coin(amount, denom);
```

Comment:
This shadows the function `coin` for the rest of this test function. I would avoid that 

(I only shadow one variable with another when transforming it, eg. `let msg = msg.to_string();`)

Reply:
I see :+1:  Renamed it to `coin1`. `allow` was not a good fit there.